src/main/java/com/kodilla/ecommercee/domain/User.java

```diff
@@ -0,0 +1,49 @@
+package com.kodilla.ecommercee.domain;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.jetbrains.annotations.NotNull;
+
+import javax.persistence.*;
+import java.util.List;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@Setter
+@Entity(name = "USER")
+public class User {
+
+    @Id
+    @GeneratedValue
+    @NotNull
+    @Column(name = "USER_ID", unique = true)
+    private Long userId;
+
+    @NotNull
+    @Column(name = "USERNAME")
+    private String username;
+
+    @NotNull
+    @Column(name = "STATUS")
+    private boolean status;
+
+    @NotNull
+    @Column(name = "USER_KEY")
+    private String userKey;
```

Comment:
Don't You think that userKey should be int or Long ?

Reply:
I changed to Long.